### DISCUSSION TITLE
Kaboom 3000 Beta Path finding function

### DISCUSSION THREAD

**Discussion by qsun81** [replit/kaboom#696] [Q&A] [answered]
is it still included in the beta version?   I thought I saw it on the changelog.md, but it is no longer there.   Also, the website and blog for the beta is down at the moment, and hopefully this would be a temporary issue.  Where else can I find the example of using the path finder?  Help me out on this, Thanks!

**Comment by lajbel** [comment, marked as answer]
The website is back! 

And yes, pathfinding is still in 3000, there's an example [kaboomjs.com](https://3000.kaboomjs.com/play?example=maze)

**Reply by qsun81** [reply]
Nice! So, it wasn't a permenant issue.  Thanks!
